### PR TITLE
OSDOCS-10335: Add missing trust policies to ROSA docs

### PR DESCRIPTION
**Pull request by bhardesty**
Version(s):
* enterprise-4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-10335

Link to docs preview:
* https://78246--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by arendej — APPROVED** [review]
lgtm

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 1]
```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
...
 ====
 
+.ROSA `ocm-role` and policy file
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-OCM-Role`
+|An IAM role that enables you to create and maintain ROSA clusters in  {cluster-manager}.
```
Comment (dfitzmau) [inline modules/rosa-sts-account-wide-roles-and-policies.adoc:668]: "Enables you" is a form of anthrophomorphism. How about:

> With this IAM role enabled, you can create and maintain ROSA clusters in {cluster-manager}.

Screenshot from 2024-07-01 12-33-44

Comment (dfitzmau) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:668]: Should a "ROSA" attribute be used here?

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 2]
```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+.ROSA `ocm-role` and policy file
```
Comment (dfitzmau) [inline modules/rosa-sts-account-wide-roles-and-policies.adoc:661]:
```suggestion
.ROSA `ocm-role` user role and policy file
```
To match other headings.

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 3]
```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
...
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-User-<OCM_user>-Role`
+|An IAM role used by Red{nbsp}Hat to verify the customer's AWS identity.
+
+|===
+
+.`sts_user_role_trust_policy.json`
```
Comment (dfitzmau) [inline modules/rosa-sts-account-wide-roles-and-policies.adoc:708]: I'm not seeing any major dfferences between this confiuration and table 7. Is this intended?

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 4]
```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
...
+                "StringEquals": {
+                    "sts:ExternalId": "<OCM_account_ID>"
+                }
+            }
+        }
+    ]
+}
+----
+====
+
```
Comment (dfitzmau) [inline modules/rosa-sts-account-wide-roles-and-policies.adoc:732]: Looks of blank lines in file? If, so remove them.

**Review by dfitzmau — APPROVED** [review]

**Comment by dfitzmau** [comment]
Looks like QE approved the PR, so please update the QE has approved this change box.

File: modules/rosa-sts-account-wide-roles-and-policies.adoc (hunk 3 above)
Comment (bhardesty) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:708]: Yes :-)

**Pull request merged**